provider/cmd/pulumi-resource-cloud-toolkit-aws/landingZone/accountIam.ts

```diff
@@ -0,0 +1,118 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import defaultsDeep from "lodash.defaultsdeep";
+import {
+  AccountIamArgs,
+  AccountPasswordPolicyArgs,
+  AccountPasswordPolicyRulesArgs,
+  defaultArgs,
+} from "./accountIamArgs";
+
+export {
+  AccountIamArgs,
+  AccountPasswordPolicyArgs,
+  AccountPasswordPolicyRulesArgs,
+  defaultArgs,
+};
+
+export class AccountIam extends pulumi.ComponentResource {
+  /**
+   * @type {AccountArgs}
+   */
+  private args: AccountIamArgs;
+
+  /**
+   * @type {string}
+   */
+  private name: string;
+
+  /**
+   * @type {aws.iam.AccountAlias}
+   */
+  public alias?: aws.iam.AccountAlias;
```

Comment:
I would set a small definition for every public parameter or function. I think avoiding empty descriptions in the schema is a good rule of thumb that might help the end user.